steps/xcode-test-bundle-builder-for-bitrise/0.0.1/step.yml

```diff
@@ -0,0 +1,73 @@
+title: Build test bundle for AWS Device Farm
```

Comment:
Drop the aws part from the title, it should work with any other tool / service, right?